applications/tari_collectibles/README.md

```diff
@@ -2,16 +2,15 @@
 
 ```bash
 # run a tari_console_wallet with grpc on port 18143
+# run a base node with grpc on port 18142
+# run a validator node with grpc on port 18144
 # ...
 
 # then in `tari_collectibles/web-app/`
 npm i
+npm run build
 
-# in `tari_collectibles/`
-cargo install tauri-cli --version "^1.0.0-beta"
-npm i
-
-npm start # npm run tauri dev
+#Then in root 
+cargo run --bin tari_collectibles
```

Comment:
We also need normal runtime instructions, i.e. not via `cargo`.
So this executable has no config or other runtime dependencies not needed by the other tari applications?